app/javascript/components/Filters/model.js

```diff
@@ -51,11 +51,19 @@ class FiltersModel {
   }
 
   constructor(initialState = {}) {
+    const { selectedQings } = initialState;
+    if (!isEmpty(selectedQings)) {
+      this.conditionSetStore.resetConditionsFromQings(selectedQings);
+
+      // eslint-disable-next-line no-param-reassign
+      delete initialState.selectedQings;
```

Comment:
If any qings should be selected, it creates new conditions for them and then deletes the original property (since it's no longer needed, it was used to create the new conditions). I've added code comments.